keyboards/7liang/tp32/v1/config.h

```diff
@@ -0,0 +1,83 @@
+
+#pragma once
+
+#define PRODUCT_ID   0x0400
+#define DEVICE_VER   0x0001
+
+#define MATRIX_ROW_PINS { B0, D1, D3 }
+#define MATRIX_COL_PINS { D4, D6, D7, B4, B5, F6, F7, B1, B2, B3, B7, D0 }
+#define UNUSED_PINS
+
+#define DIODE_DIRECTION COL2ROW
+
+#define DEBOUNCE 5
+
+#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_RESYNC_ENABLE
+
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+#define RGB_DI_PIN          F0
+#define RGBLED_NUM          32
+#define RGBLIGHT_ANIMATIONS

```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```